npm-shrinkwrap.json

```diff
@@ -190,6 +190,12 @@
         "lodash": {
           "version": "3.10.1"
         },
+        "lodash.assign": {
```

Comment:
Super minor, but now our build is shipping `lodash/assign` and `lodash.assign`. Unless there is a reason to switch, we might want to use the prior (which we use in Calypso) in related a8c packages like `i18n-calypso`.